packages/services/schema/src/lib/__tests__/add-inaccessible-to-unreachable-types.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { parse } from 'graphql';
+import { extractLinkImplementations } from '@graphql-hive/federation-link-utils';
+import { addInaccessibleToUnreachableTypes } from '../add-inaccessible-to-unreachable-types';
+import { composeFederationV2 } from '../compose';
+
+test('Filters based on tags', async () => {
```

Comment:
i know we have integration tests covering this stuff. But personally I find unit tests to be a lot quicker to write and run.